mapit_gb/management/commands/mapit_UK_show_missing_codes.py

```diff
@@ -1,9 +1,24 @@
 from django.core.management.base import NoArgsCommand
 from mapit.models import Area, Generation
+from optparse import make_option
 
 
 class Command(NoArgsCommand):
     help = "Prints results of queries to check we aren't missing SNAC and GSS codes"
+    option_list = NoArgsCommand.option_list + (
+        make_option(
+            "--code_types", dest="code_types",
+            help="The list of code types to search for",
+            default="ons,gss"),
```

Comment:
@jennyd @h-lame I've made some changes to the `mapit_UK_show_missing_codes` command, basically ignoring the `govuk_slug` code_type for area_type `EUR`.